node_modules/.package-lock.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Same comment as the other package-lock files for why this is here and checked in?